packages/mdc-drawer/README.md

```diff
@@ -234,6 +251,35 @@ topAppBar.listen('MDCTopAppBar:nav', () => {
 });
 ```
 
+### Modal Drawer
+
+Modal drawers are elevated above most of the app’s UI and don’t affect the screen’s layout grid.
+
+```html
+<body>
+  <header class="mdc-drawer mdc-drawer--modal">
+    <div class="mdc-drawer__content">
+      <nav class="mdc-list">
+        <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">inbox</i>Inbox
+        </a>
+        <a class="mdc-list-item" href="#">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">send</i>Outgoing
+        </a>
+        <a class="mdc-list-item" href="#">
+          <i class="material-icons mdc-list-item__graphic" aria-hidden="true">drafts</i>Drafts
+        </a>
+      </nav>
+    </div>
+  </header>
+
+  <div class="mdc-drawer-scrim"></div>
+  <div>Main Content</div>
+</body>
+```
+
+> Use the `mdc-drawer-scrim` class on next sibling element to add backdrop to block app's UI.
```

Comment:
I think backdrop might potentially mean something else in Material terms...

Maybe:

```
> The `mdc-drawer-scrim` next sibling element protects the app's UI from interactions while the drawer is open.
```